src/app/services/cognito.service.ts

```diff
@@ -113,10 +115,38 @@ export class CognitoService {
         resultStream.next(err);
       }
     });
-
     return resultStream;
   }
+
+/*
+  * The user will need to provide their email which Cognito for check the user
+  * pool for and then create a new password.
+  */
```

Comment:
add author and params
you can use /** to generate NgDocs

Reply:
Completed.